### PR TITLE
Template updates after feedback

### PR DESCRIPTION
**Pull request by choldgraf**
Some more feedback from OceanHackWeek!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-hub.yml [hunk 1]
```diff
@@ -83,26 +92,18 @@ body:
...
-      label: Extra features you'd like to enable
-      description: |
-        In addition to a base JupyterHub, you may enable extra infrastructure features that provide additional functionality. These generally come with added complexity and cost, so make sure you've discussed these options with somebody at 2i2c before selecting them here.
-
+        What kind of authentication service will be used for this community. This is the type of "log-in username" users will use on the hub.
       options:
-        - label: Dedicated Scalable Dask Cluster with [Dask Gateway](https://gateway.dask.org/)
+        - GitHub Authentication (e.g., @mygithubhandle)
+        - Google OAuth (e.g., myemailaddress@2i2c.org)
+        - Other (may not be possible, please specify in comments)
```
Comment (yuvipanda) [inline .github/ISSUE_TEMPLATE/new-hub.yml:104]:
```suggestion
        - Google OAuth (e.g., myemailaddress@2i2c.org)
        - ORCID
        - Other (may not be possible, please specify in comments)
```

**Comment by yuvipanda** [comment]
OK to merge once the merge conflict is resolved!

**Review by damianavila — APPROVED** [review]
LGTM (modulo the merge conflict are resolved).

**Comment by choldgraf** [comment]
merging this in since we've got approvals, thanks all :-)

**Pull request merged**